src/components/Conversation/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,20 @@
+import { UserRepresentation } from "@/types/users.types";
+
+type AvatarProps = {
+    user: UserRepresentation;
+};
+
+export default function Avatar({ user }: AvatarProps) {
```

Comment:
I'm assuming the creation of this component is in reference to #9 's comment to use Mantine's Avatar component. By that, I meant to use the [Avatar component](https://mantine.dev/core/avatar/) from the component library [Mantine](https://mantine.dev)